sample/build.gradle

```diff
@@ -22,9 +23,13 @@ android {
 }
 
 dependencies {
+  compile fileTree(dir: 'libs', include: ['*.jar'])
```

Comment:
You don't need this line, do you?

Reply:
uhm.... i guess it was automatically changed, because i don't remember adding or updating it

Reply:
Done @SmasSive